tests/test_cli_download_files.py

```diff
@@ -68,6 +68,23 @@ def test_download_files():
         os.remove(test_file)
 
 
+def test_download_files_root():
```

Comment:
(Well, you could actually also amend test behaviour so that it would not result in E if xrootd is not available, and only test root downloads if xrootd is present.)

Reply:
Done 👍

Reply:
I tried to implement the test, pushed to check if it works on `gh-actions`!